chain/jsonrpc/src/lib.rs

```diff
@@ -386,6 +386,9 @@ impl JsonRpcHandler {
         Ok(match request.method.as_ref() {
             // Handlers ordered alphabetically
             "block" => process_method_call(request, |params| self.block(params)).await,
+            "block_effects" => {
```

Comment:
Instead of duplicating the code, I would use `|` matching:

```rust
match method {
    "block_effects" | "EXPERIMENTAL_changes_in_block" => {
        ...
    }
}
```

This would better indicate that this is actually just an alias to the existing name.